.github/workflows/ci.yml

```
@@ -0,0 +1,27 @@
+          version: 8
+      - name: Use Node.js 20.x
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          cache: 'pnpm'
+      - name: Install dependencies
+        run: pnpm install
+      - name: Lint
+        run: pnpm lint
+      - name: Test
+        run: pnpm test 
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)